fineract-doc/src/docs/en/chapters/development/index.adoc

```diff
@@ -6,6 +6,8 @@ include::git.adoc[leveloffset=+1]
 
 include::gpg.adoc[leveloffset=+1]
 
+include::instance-type.adoc[leveloffset=+1]
```

Comment:
I think this section should rather be in the architecture one (or maybe deployment). Thoughts?

Reply:
Yeah, it could be any of them, but I would vote for deployment rather than architecture.